<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="Sandy" failures="1" tests="1" name="testRunner.Runtest" time="12.192" errors="0" timestamp="2024-05-26T21:38:41 IST" skipped="0">
  <testcase classname="testRunner.Runtest" name="runScenario" time="12.192">
    <failure message="expected [false] but found [true]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: expected [false] but found [true]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1578)
at org.testng.Assert.assertEqualsImpl(Assert.java:150)
at org.testng.Assert.assertEquals(Assert.java:132)
at org.testng.Assert.assertEquals(Assert.java:1067)
at org.testng.Assert.assertEquals(Assert.java:1031)
at org.testng.Assert.assertEquals(Assert.java:1077)
at stepDefintions.Steps.user_should_find_customer_name_info_in_search_list(Steps.java:172)
at âœ½.user should find customer name info in search list(file:///C:/Users/sande/eclipse-workspace/CucumberPracticeYTpavan/src/test/java/java/Feature/Customer.feature:31)
]]>
    </failure>
  </testcase> <!-- runScenario -->
</testsuite> <!-- testRunner.Runtest -->
